data/publiccloud/terraform/sap/azure.tfvars

```diff
@@ -131,27 +194,74 @@ reg_code = "%SCC_REGCODE_SLES4SAP%"
+drbd_ips = ["10.74.1.21", "10.74.1.22"]
+drbd_cluster_vip = "10.74.1.23"
+
+# Enable SBD for the drbd cluster
+drbd_cluster_sbd_enabled = true
+
 # Netweaver variables
 
 #netweaver_enabled = true
+# If the addresses are not set they will be auto generated from the provided vnet address range
 #netweaver_ips = ["10.74.1.30", "10.74.1.31", "10.74.1.32", "10.74.1.33"]
 #netweaver_virtual_ips = ["10.74.1.35", "10.74.1.36", "10.74.1.37", "10.74.1.38"]
+
+# Enabling this option will create a ASCS/ERS HA available cluster
+#netweaver_ha_enabled = true
+# Application servers number. The 1st one will be a PAS instance, and the rest AAS servers
+#netweaver_app_server_count = 2
+
+# VM sizes
+#netweaver_xscs_vm_size = Standard_D2s_v3
+#netweaver_app_vm_size = Standard_D2s_v3
+
+# Set the Netweaver product id for HA (this is just an example)
+#netweaver_product_id = NW750.HDB.ABAPHA
+# Fon non HA
+#netweaver_product_id = NW750.HDB.ABAP
+
+# Enable SBD for the netweaver cluster
+#netweaver_cluster_sbd_enabled = true
+
+# This storage account must contain the next software (select the version you want to install of course)
+#SWPM - `IND:SLTOOLSET:2.0:SWPM:*:LINUX_X86_64:*x`
+#Netweaver export - `SAP:NETWEAVER:750:DVD_EXPORT:SAP NetWeaver 750 Installation Export DVD 1/1:D51050829_2`
+#HANA Platform- `HDB:HANA:2.0:LINUX_X86_64:SAP HANA PLATFORM EDITION 2.0::BD51053787`
+#Sapexe folder files:
+#igsexe_23-20007790.sar  igshelper_4-10010245.sar  SAPEXE_501-80002573.SAR  SAPEXEDB_501-80002572.SAR  SAPHOSTAGENT45_45-20009394.SAR
+
 #netweaver_storage_account_key = "YOUR_STORAGE_ACCOUNT_KEY"
 #netweaver_storage_account_name = "YOUR_STORAGE_ACCOUNT_NAME"
 #netweaver_storage_account = "//YOUR_STORAGE_ACCOUNT_NAME.file.core.windows.net/path/to/your/nw/installation/master"
+# Netweaver installation required folders
+# SAP SWPM installation folder, relative to the netweaver_storage_account mounting point
+#netweaver_swpm_folder     =  "your_swpm"
+# Or specify the path to the sapcar executable & SWPM installer sar archive, relative to the netweaver_storage_account mounting point
+# The sar archive will be extracted to path specified at netweaver_extract_dir under SWPM directory (optional, by default /sapmedia/NW/SWPM)
+#netweaver_sapcar_exe = "your_sapcar_exe_file_path"
+#netweaver_swpm_sar = "your_swpm_sar_file_path"
+# Folder where needed SAR executables (sapexe, sapdbexe) are stored, relative to the netweaver_storage_account mounting point
+#netweaver_sapexe_folder   =  "kernel_nw75_sar"
+# Additional media archives or folders (added in start_dir.cd), relative to the netweaver_storage_account mounting point
+#netweaver_additional_dvds = ["dvd1", "dvd2"]
 
 # QA variables
 
 # Define if the deployment is using for testing purpose
 # Disable all extra packages that do not come from the image
 # Except salt-minion (for the moment) and salt formulas
-# true or false
-qa_mode = true
```

Comment:
just out of curiosity can you elaborate about this change ? because if testing profile is not using qa_mode than why we need it for ?

Reply:
`qa_mode` is not used anymore in the TF code and will be removed in the next version (confirmed by the devs).

*Note:* the TF code is not developed by us (qa-asg).

Reply:
Yes, the main project we are testing is this [one](https://github.com/SUSE/ha-sap-terraform-deployments).